src/movement/control_system.cpp

```diff
@@ -187,265 +195,355 @@ void ControlSystem::InputControlMessage(robosub::control msg)
                 this->goal_types[i] = robosub::control::STATE_ABSOLUTE;
                 this->goals[i] = control_values[i];
                 break;
+
+            /*
+             * Note that all rotational goals must be wrapped according to the
+             * following bounding limits:
+             *   Roll: (-180, 180)
+             *   Pitch: (-90, 90)
+             *   Yaw: (-180, 180)
+             */
             case robosub::control::STATE_RELATIVE:
                 this->goal_types[i] = robosub::control::STATE_ABSOLUTE;
```

Comment:
Why is this set to `STATE_ABSOLUTE` when the requested state is `STATE_RELATIVE`?